Release v3.3.0 (#86)
appcircleio/appcircle-docusaurus: docs/updates/release-notes.md

# Latest Release Notes

## 3.3.0 - 2023-04-27 - Data Theorem Mobile Secure, App Center CodePush

### 🆕 New Feature
- [Data Theorem Mobile Secure](../workflows/common-workflow-steps.md#data-theorem-mobile-secure) workflow step added. <CloudBadge/> <SelfHostedBadge/>
- [App Center CodePush](../workflows/react-native-specific-workflow-steps.md#app-center-codepush) workflow step added. <CloudBadge/> <SelfHostedBadge/>
- Latest five build status added to build profile. <CloudBadge/> <SelfHostedBadge/>

> - [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/> <SelfHostedBadge/>

Reviewer: Currently slack bot is not supported on self-hosted. It's in future roadmap.
```suggestion
- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/>
```

### :muscle: Improvement
- SVG images are updated <CloudBadge/> <SelfHostedBadge/>
- Build profile card design is improved. <CloudBadge/> <SelfHostedBadge/>

> - Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>

Reviewer: To mention general UI improvements done in April.

```suggestion
- Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>
- Lots of UI and text improvements were made for better UX. <CloudBadge/> <SelfHostedBadge/>
```

### 🐞 Fixed

> - [Huawei AppGallery](../store-submit/huawei-app-gallery.md) submission bug fixed.  <SelfHostedBadge/>

Reviewer: :warning: We need to wait for the discussion to be resolved for this item. See details in [here](https://appcircleteam.slack.com/archives/C02RXLSMU3F/p1682693627798339).

Author: It's fixed today.

> - [Enterprise Store](../enterprise-appstore/customize-ent-store.md) 2FA localization bug fixed. <SelfHostedBadge/>

Reviewer: @tosbaha I could not find any issue for this item in April roadmap.

We can replace it with below note. (BE-1256)
```suggestion
- [Enterprise Store](../enterprise-appstore/customize-ent-store.md) The background image bug was fixed on the login page. <SelfHostedBadge/>
```
